library/pdf_creator.class.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * PdfCreator class
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Jerry Padgett <sjpadgett@gmail.com>
+ * @copyright Copyright (c) 2017 Jerry Padgett <sjpadgett@gmail.com>
+ * @license https://www.gnu.org/licenses/agpl-3.0.en.html GNU Affero General Public License 3
+ */
```

Comment:
Place the following (needs to be above the use command and it needs an empty line both above and below it):
``````
namespace OpenEMR\Pdf;
``````